lang/js/suggested-app-structure.md

```diff
@@ -0,0 +1,27 @@
+Suggested App Structure
+=======================
+
+The below is an example application structure. This application uses HTTP as a communication layer but it would work equally well with other communication types.
+
+	├── bin
+	├── doc
+	├── http
+	│   ├── lib
+	│   └── route
+	├── lib
+	└── spec
+
+* `bin` - Executable files. These deal with a command line interface as the communication layer. If the executable must use something in another communication layer (for example starting a server for HTTP) add as little from that layer as possible.
```

Comment:
Command Line Interface should be capitalized and put (CLI) afterwards, just in case you use CLI later on.